design/FY2017/NFP-SingleDuctAirTerminalConstantVolumeNoReheat.md

```diff
@@ -0,0 +1,212 @@
+
+
+## New idf object based on the proposed IDD object.
+
+#### AirTerminal:SingleDuct:ConstantVolume:NoReheat New Object
+
+```
+AirTerminal:SingleDuct:ConstantVolume:NoReheat,
+
+      NoReheat Zone 1,                !- Name of System
+      AlwaysOnFanAvailSched,          !- Availability Schedule Name
+      Zone 1 Unit Air Outlet Node,    !- Air Outlet Node Name
+      Zone 1 Unit Air Inlet Node,     !- Air Inlet Node Name
+      0.60,                           !- Maximum Air Flow Rate {m3/s}
+      ,                               !- Design Specification Outdoor Air Object Name
+      ;                               !- Per Person Ventilation Rate Mode
+
+
+```
+
+## Outputs Description ##
+
+  HVAC,Sum,Zone Air Terminal Sensible Heating Energy {[}J{]}
+
+  HVAC,Sum,Zone Air Terminal Sensible Cooling Energy {[}J{]}
+
+  HVAC,Average,Zone Air Terminal Sensible Heating Rate {[}W{]}
+
+  HVAC,Average,Zone Air Terminal Sensible Cooling Rate {[}W{]}
+
+
+Zone Air Terminal Sensible Heating Energy 
+
+Zone Air Terminal Sensible Heating Rate
+
+These outputs are the sensible heating energy and heating rate provided to the zone by the air terminal single duct constant volume no reheat object.
+
+Zone Air Terminal Sensible Cooling Energy
+
+Zone Air Terminal Sensible Cooling Rate
+
+These outputs are the sensible cooling energy and cooling rate provided to the zone by the air terminal single duct constant volume no reheat object.
+
+## Engineering Reference ##
+
+The input object AirTerminal:SingleDuct:ConstantVolume:NoReheat is used to pass conditioned or treated central supply air directly into a zone without any reheat. This terminal air equipment allows to supply central system air directly to a zone without any zone level control or tempering. The supply air temperature is controlled by the central system to meet the load in a controlled zone. This object is configured for use with a constant volume central air system or Furnace and variable supply air temperature. This unit allows the program to know what zone this branch of the air system is attached to, air inlet and outlet nodes, an input field for the maximum air flow rate, and other two optional input fields. The air inlet node should be the same as one of the AirLoopHVAC:ZoneSplitter object outlet nodes. The air outlet node name should be same as zone air inlet node name and the air distribution unit air outlet node name. The last two input fields: *Design Specification Outdoor Air Object Name*, and *Per Person Ventilation Rate Mode* are used for calculating the outdoor air requirement of an air terminal unit. This component is typically used with an AirLoopHVAC running as a constant-volume and variable-temperature system.
```

Comment:
Make it clear hear that the DSOA sets the terminal unit flow rate.  I think you can delete the last sentence.